docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,576 @@
+    "    This function returns a dictionary mapping multi-labels to unique colors\n",
+    "    \"\"\"\n",
+    "    dcolors ={'[0]': 'aa4400',\n",
+    "             '[0, 2]': '55227f',\n",
+    "             '[0, 1]': '55a100',\n",
+    "             '[1]': '00ff00',\n",
+    "             '[1, 2]': '007f7f',\n",
+    "             '[0, 1, 2]': '386b55',\n",
+    "             '[2]': '0000ff'}\n",
+    "\n",
+    "    return [\"#\"+dcolors[str(i)] for i in labels]\n",
+    "\n",
+    "def plot_data(data, circles, title, alpha=1.0,colors = []):\n",
+    "    assert colors is not None\n",
+    "    plt.figure(figsize=(14, 5))\n",
+    "    done = set()\n",
+    "    for i in range(0,len(data)):\n",
+    "        lab = str(labels[i])\n",
+    "        if lab in done:\n",
+    "            label = \"\"\n",
+    "        else:\n",
+    "            label = lab\n",
+    "            done.add(lab)\n",
+    "        plt.scatter(data[i, 0], data[i, 1], c=colors[i], s=30,alpha=0.6, label = label)\n",
+    "    for i in circles:\n",
+    "        plt.plot(\n",
+    "            data[i][0],\n",
+    "            data[i][1],\n",
+    "            \"o\",\n",
+    "            markerfacecolor=\"none\",\n",
+    "            markeredgecolor=\"red\",\n",
+    "            markersize=14,\n",
+    "            markeredgewidth=2.5,\n",
+    "            alpha=alpha\n",
+    "        )\n",
+    "    _ = plt.title(title, fontsize=25)\n",
+    "    plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "672bfc2a",
+   "metadata": {},
+   "source": [
+    "Some of the labels in our generated dataset purposely contain errors. The examples with label errors are circled in the plot below, which depicts the dataset. This dataset contains 3 classes, and any subset of these may be the given label for a particular example. We say this example has a label error if it is better described by an alternative  subset of the classes than the given label."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dac65d3b-51e8-4682-b829-beab610b56d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_class = 3\n",
+    "dataset = make_multilabel_data()\n",
+    "labels = dataset['labels']\n",
+    "true_errors = np.where(np.sum(int2onehot(dataset['true_labels_train'],3)!=int2onehot(dataset['labels'],3),axis=1)>=1)[0]\n",
+    "plot_data(dataset['X_train'], circles=true_errors, title=f\"True label errors in multi-label dataset with {num_class} classes\", colors = get_color_array(labels),alpha=0.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "144ad4c2-49bb-4147-a743-a83ed1656a11",
+   "metadata": {},
+   "source": [
+    "## 2. Format data, labels, and model predictions\n",
+    "\n",
+    "In multi-label classification, each example in the dataset is labeled as belonging to one **or more** of *K* possible classes. To find label issues, cleanlab requires predicted class probabilities from a trained classifier. \n",
```

Comment:
In principle, this should be "zero or more".

Reply:
good catch! clarified